.github/workflows/publish_docs_to_wiki.yml

```diff
@@ -0,0 +1,36 @@
+name: Publish docs to Wiki
+
+on:
+  push:
+    paths:
+      - docs/**
+    branches:
+      - main
+      
+env:
+  USER_TOKEN: ${{ secrets.WIKI_ACTION_TOKEN }}
+  USER_NAME: team-nimblehq
+  USER_EMAIL: dev@nimblehq.co
+  OWNER: ${{ github.event.repository.owner.name }}
+  REPOSITORY_NAME: ${{ github.event.repository.name }}
+
+jobs:
+  Publish-Docs-To-Wiki:
+    runs-on: ubuntu-latest
+    steps:  
+    - uses: actions/checkout@v2
+    - name: Prepare
+      run: |
+         mkdir tmp_wiki
+         cd tmp_wiki
+         git init
+         git config user.name $USER_NAME
+         git config user.email $USER_EMAIL
+         git pull https://$USER_TOKEN@github.com/$OWNER/$REPOSITORY_NAME.wiki.git
+    - name: Push
```

Comment:
also the line break between steps 🙇  you may want to ref from here: https://nimblehq.co/compass/development/code-conventions/github-actions/

since we had the convention for this, we may have to refactor the other workflow later as well 😂